apps/site/assets/css/_mixins.scss

```diff
@@ -317,3 +325,41 @@
     top: .125 * $width-height;
   }
 }
+
+// Make container with list of items appear as a horizontally
+// scrolling section rather than stacked, at a certain breakpoint.
+@mixin side-scroll($breakpoint) {
+  @include media-breakpoint-down($breakpoint) {
+    // scss-lint:disable all
+    display: flex;
+    overflow-x: auto;
+
+    // reset overrides
+    flex-wrap: nowrap;
+    justify-content: normal;
+    margin: 0;
+    padding: 0;
+
+    // Hide default horizontal scrollbar
```

Comment:
You would not [likely] be on a desktop computer at the breakpoints where these styles take effect. You would [likely] be on a mobile or other touch-enabled device, which do not show scrollbars anyway.

For most Mac or other laptop users, horizontal scroll on the mouse or trackpad works.

Reply:
> You would [likely] be on a mobile or other touch-enabled device, which do not show scrollbars anyway.

Ok — then maybe we don't need to bother disabling the scrollbars, since they'll already be hidden? I'm just wary of removing a built-in browser feature without providing an alternative, even if it might look weird under some uncommon conditions.